tensorboard/webapp/runs/views/runs_table/sorting_utils.ts

```diff
@@ -0,0 +1,131 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {
+  SortingInfo,
+  SortingOrder,
+  TableData,
+} from '../../../widgets/data_table/types';
+import {ExperimentAlias} from '../../../experiments/types';
+
+enum UndefinedStrategy {
+  BEFORE,
+  AFTER,
+}
+
+interface SortOptions {
```

Comment:
Why do we need this interface with a single attribute. Is there a reason you cannot just use the enum directly everywhere this interface is used?

Reply:
This is a pretty complicated function and I thought that using an object as a parameter would make the options provided more readable.

Reply:
I am ok with that justification. I think I would prefer naming the enum something more readable....UndefinedValuePlacement?

I cant really think of something good. Maybe this is best.